lib/cartopy/crs.py

```diff
@@ -1815,6 +1815,59 @@ def y_limits(self):
         return self._y_limits
 
 
+class LambertZoneII(Projection):
+    """
+    Lambert zone II (extended) projection (https://epsg.io/27572).
```

Comment:
Could you add some more text about this being used for France or general information about this projection so someone reading this would know more at first glance?

Reply:
I have added slightly more information in the docstring.